apps/opencs/model/world/columnimp.hpp

```diff
@@ -2417,7 +2434,125 @@ namespace CSMWorld
             return true;
         }
     };
-    
+
+    template<typename ESXRecordT>
```

Comment:
I don't understand this whole section. Why are these templates?

Reply:
I made them templates because that was what I observed the other columns doing (even for non-generic data), and I assumed there was a reason for it. For example, mUseValue is only in one esm structure (a class called Skill), but also has a template based column.

Reply:
Hm, you are right. We have almost all columns as templates. I started out this way because I didn't know in advance which columns I would have to re-use for other record types. Probably a bit wasteful. Should have looked more closely and avoided making columns templates that clearly only belong to one table.
Nvm, then. We can look at that another time as part of a general code review.